### PR TITLE
Add Mutation [MUTA] axis 

### PR DESCRIPTION
**Pull request by vv-monsalve**
PR to follow up on the inclusion of the new custom axis Mutation `MUTA` #9 

Required for the upcoming font Kablammo.

### CODE REVIEW

**Review by chrissimpkins — APPROVED** [review]
LGTM.  I asked Sarah and Dave to review the description.  OK to merge so that we can begin the axis registration process and onboard the Kablammo family into sandbox for testing.

**Comment by chrissimpkins** [comment]
I'll take a look at the CI fails.  They are related to the changes in #89

File: Lib/axisregistry/data/mutation.textproto [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# from Kablammo 
+tag: "MUTA"
```
Comment (davelab6) [inline Lib/axisregistry/data/mutation.textproto:2]: Needs project URL

**Comment by chrissimpkins** [comment]
rebased on CI fixes that I merged into main

File: Lib/axisregistry/data/mutation.textproto (hunk 1 above)
Comment (chrissimpkins) [reply Lib/axisregistry/data/mutation.textproto:2]: > https://github.com/google/fonts/issues/3802#issue-987769918

Private?

Comment (davelab6) [reply Lib/axisregistry/data/mutation.textproto:2]: Release the kraken

**Comment by chrissimpkins** [comment]
Merged so that we can begin the registration process. @vv-monsalve please reach out to the development team to open the repository and add the repository URL in a header comment in a new PR. Thanks!

**Comment by chrissimpkins** [comment]
https://github.com/google/fonts/pull/5704

**Comment by vv-monsalve** [comment]
MUTA was replaced by #120

**Pull request merged**